src/java.desktop/macosx/classes/sun/lwawt/macosx/CPrinterJob.java

```diff
@@ -251,6 +253,22 @@ private void setPrintToFile(boolean printToFile) {
         isPrintToFile = printToFile;
     }
 
+    private void setDestinationFile(String dest) {
+        System.out.println("dest="+dest+" attrs="+attributes);
```

Comment:
Probably this println should be removed.

Reply:
oops